.azure/pipelines/ci.yml

```diff
@@ -655,15 +657,19 @@ stages:
 
   - template: jobs/default-build.yml
     parameters:
-      condition: ne(variables['Build.Reason'], 'PullRequest')
+      # condition: ne(variables['Build.Reason'], 'PullRequest') UNCOMMENT after testing
```

Comment:
Is this bit ready to review?

Reply:
Yes, the only way to test the daily/arm queues is by commenting out the conditions, I'll undo them once everything is ready to go